src/api/api.ts

```diff
@@ -73,3 +74,22 @@ export function hentFremtidigSituasjonList(): Promise<Array<HovedmalType&Histori
 export function oppdaterMal(mal: string): Promise<MalType> {
     return fetchData<MalType>(`${API_VEILARBOPPFOLGING}/mal`, {method: 'post', body: JSON.stringify({mal}), ...CONFIG});
 }
+
+export function oppdaterHelseHinder(hinder: boolean): Promise<HensynType> {
```

Comment:
Burde bare være en api metode i backend som henter Fremtidigsituasjon, Helse utfordring, og Andre utfordringer.
Istede for 3 forskjellige

Reply:
dette er en funksjon for og oppdatere ikke hente